### PR TITLE
Add air-gapped install docs

### PR DESCRIPTION
**Pull request by kilfoyle**
This ports the [air-gapped install doc](https://docs.google.com/document/d/1hR9C9661-QpYZuy1IASzDaQoY0Ks2eU7NCE4ok3tOdk/edit#heading=h.930xdqhukf3f) compiled by SAs into the Installation and Upgrade Guide section of the public docs.

CC: @jstibbards-elastic & @KOTungseth as you'd both expressed interest in these.

[PREVIEW SITE](https://stack-docs_2458.docs-preview.app.elstc.co/guide/en/elastic-stack/master/air-gapped-install.html)

---

Screenshot 2023-07-10 at 3 44 35 PM

Closes: https://github.com/elastic/platform-docs-team/issues/73

### CODE REVIEW

**Comment by jstibbards-elastic** [comment]
Thank you!

- James

File: docs/en/install-upgrade/air-gapped-install.asciidoc [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+
+[source,shell]
+----
+#!/usr/bin/env bash
+
+EPR_BIND_ADDRESS="0.0.0.0"
+EPR_BIND_PORT="8443"
+EPR_TLS_CERT="/etc/elastic/epr/epr.pem"
+EPR_TLS_KEY="/etc/elastic/epr/epr-key.pem"
+EPR_IMAGE="docker.elastic.co/package-registry/distribution:8.4.3"
```
Comment (philippkahr) [inline docs/en/install-upgrade/air-gapped-install.asciidoc:208]: This contains a fixed version number. Can we make this to the same one as the page you are viewing?

File: docs/en/install-upgrade/air-gapped-install.asciidoc [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+	--sdnotify=conmon \
+	-d \
+	--replace \
+	--name elastic-epr \
+	-p 0.0.0.0:8443:8443 \
+	-v /etc/elastic/epr/epr.pem:/etc/ssl/epr.crt:ro \
+	-v /etc/elastic/epr/epr-key.pem:/etc/ssl/epr.key:ro \
+	-e EPR_ADDRESS=0.0.0.0:8443 \
+	-e EPR_TLS_CERT=/etc/ssl/epr.crt \
+	-e EPR_TLS_KEY=/etc/ssl/epr.key docker.elastic.co/package-registry/distribution:8.6.2
```
Comment (philippkahr) [inline docs/en/install-upgrade/air-gapped-install.asciidoc:257]: This contains a fixed version number. Can we make this to the same one as the page you are viewing?

File: docs/en/install-upgrade/air-gapped-install.asciidoc [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+=== Appendix B - {artifact-registry}
+
+The following example script downloads artifacts from the internet to be later served as a private Elastic Package Registry.
+
+[source,shell]
+----
+#!/usr/bin/env bash
+set -o nounset -o errexit -o pipefail
+
+STACK_VERSION=8.4.3
```
Comment (philippkahr) [inline docs/en/install-upgrade/air-gapped-install.asciidoc:278]: Same here fixed version number

File: docs/en/install-upgrade/air-gapped-install.asciidoc [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+      app: elastic-package-registry
+  template:
+    metadata:
+      name: elastic-package-registry
+      labels:
+        app: elastic-package-registry
+    spec:
+      containers:
+        - name: epr
+          image: docker.elastic.co/package-registry/distribution:8.6.1
```
Comment (philippkahr) [inline docs/en/install-upgrade/air-gapped-install.asciidoc:409]: Fixed version number

**Review by philippkahr — COMMENTED** [review]
Looks good to me. Fixed version numbers should be fixed.

File: docs/en/install-upgrade/air-gapped-install.asciidoc (hunk 4 above)
Comment (kilfoyle) [reply docs/en/install-upgrade/air-gapped-install.asciidoc]: Fixed in [40dbeb3](https://github.com/elastic/stack-docs/pull/2458/commits/40dbeb3c10d7aa1d5314b996b9c13f4a0acf5e9c)

File: docs/en/install-upgrade/air-gapped-install.asciidoc (hunk 3 above)
Comment (kilfoyle) [reply docs/en/install-upgrade/air-gapped-install.asciidoc]: [40dbeb3](https://github.com/elastic/stack-docs/pull/2458/commits/40dbeb3c10d7aa1d5314b996b9c13f4a0acf5e9c)

File: docs/en/install-upgrade/air-gapped-install.asciidoc (hunk 2 above)
Comment (kilfoyle) [reply docs/en/install-upgrade/air-gapped-install.asciidoc]: Fixed in [40dbeb3](https://github.com/elastic/stack-docs/pull/2458/commits/40dbeb3c10d7aa1d5314b996b9c13f4a0acf5e9c)

**Review by nimarezainia — APPROVED** [review]

**Pull request merged**